src/test/java/homeworks/POM/Pages/LoginPageHmw.java

```diff
@@ -1,36 +1,41 @@
 package homeworks.POM.Pages;
 
-import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.FindBy;
 
-public class LoginPageHmw extends BasePageHmw{
-By emailFieldLocator = By.cssSelector("[type='email']");
-By passwordFieldLocator = By.cssSelector("[type='password']");
-By submitButtonLocator = By.cssSelector("[type='submit']");
 
+public class LoginPageHmw extends BasePageHmw{
+@FindBy(css="[type='email']")
+        WebElement emailField;
+@FindBy (css="[type='password']")
+        WebElement passwordField;
+@FindBy (css="[type='submit']")
+WebElement submitButton;
```

Comment:
**I recommend:  make all the locators and WebElements private,** because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example


```
  @FindBy(css = "[type='email']")
  private WebElement emailField;
  @FindBy(css = "[type='password']")
  private WebElement passwordField;
  @FindBy(css = "[type='submit']")
  private WebElement submitButton;
```